generator/src/main/java/com/scottlogic/deg/generator/Guice/FileOutputTargetProvider.java

```diff
@@ -2,28 +2,29 @@
 
 import com.google.inject.Inject;
 import com.google.inject.Provider;
-import com.scottlogic.deg.generator.CommandLine.GenerateCommandLine;
+import com.scottlogic.deg.generator.generation.GenerationConfigSource;
 import com.scottlogic.deg.generator.outputs.dataset_writers.CsvDataSetWriter;
 import com.scottlogic.deg.generator.outputs.dataset_writers.DataSetWriter;
 import com.scottlogic.deg.generator.outputs.dataset_writers.MultiDataSetWriter;
 import com.scottlogic.deg.generator.outputs.dataset_writers.SourceTracingDataSetWriter;
+import com.scottlogic.deg.generator.outputs.targets.DirectoryOutputTarget;
 import com.scottlogic.deg.generator.outputs.targets.FileOutputTarget;
 
 public class FileOutputTargetProvider implements Provider<FileOutputTarget> {
```

Comment:
similar to above comment, this seems very similar to the `DirectoryOutputTargetProvider `, could there be one provider to return either based on a command line argument?